scripts.py

```diff
@@ -1088,7 +1088,7 @@ def explain(self, licensepool):
         ]
         data.append(", ".join(title_removals))
 
-        print("\t".join([str(x).encode("utf8") for x in data]))
+        print("\t".join([str(x) for x in data]))
```

Comment:
I'm not sure why this fixes the problem but it does.   I'm seeing titles and authors with UTF-8 characters.  And the stacktrace I was getting doesn't make sense it me.  I think there must be something in the way that pythons handles strings that I don't understand.

Reply:
I have no idea how this code was ever valid.

```
>>> "a".join("b")
'b'
>>> "a".join("b".encode("utf-8"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: sequence item 0: expected str instance, int found
```

Maybe old Python versions were lax about strings vs bytes?

Reply:
This must be leftover from the Python 2 -> Python 3 migration. The biggest difference between Python 2 and Python 3 is that strings were bytes in Python 2, and there was a separate unicode type. 

https://python-future.org/compatible_idioms.html#strings-and-bytes
http://python3porting.com/problems.html#bytes-strings-and-unicode

Most of these bytes vs string issues have been sorted out by now, although there is still some conversion code that could be removed now that we are Python 3 only.